src/version.txt

```diff
@@ -1 +1 @@
-__version__ = "1.0.0-rc.2"
+__version__ = "1.0.0"
```

Comment:
We typically don't finalize the version until after a prerelease AMI has been created and tested.  Are you planning to do that for this PR?

Reply:
After chatting with @itsmostafa, this convo can be resolved.  A prerelease AMI will be created and tested.